ecosystem/sep-0010.md

```diff
@@ -112,19 +112,21 @@ GET https://auth.example.com/?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDI
 On success the endpoint must return `200 OK` HTTP status code and a JSON object with these fields:
 
 * `transaction`: an XDR-encoded Stellar transaction with the following:
-  * source account set to server's signing account
+  * source account set to the **Server Account**
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
-  * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
+  * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give the **Client** time to sign transaction)
   * operations:
-    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-      * The value of key is the home domain the client is authenticating with, followed by `auth`. It can be at most 64 characters.
+    * `manage_data(source: client account, key: '<home domain> auth', value: random_nonce())`
+      * The source account is the **Client Account**
+      * The value of key is the **Home Domain**, followed by `auth`. It can be at most 64 characters.
       * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
     * subsequent operations, order unimportant:
-      * `manage_data(source: server_account, key: 'web_auth_domain', value: web_auth_domain)`
-        * The value is the web auth domain, the domain hosting the web auth endpoint and that the client connected to when requesting the challenge transaction. It can be at most 64 characters.
-      * zero or more `manage_data(source: server_account, ...)` reserved for future use
-  * signature by the service's stellar.toml `SIGNING_KEY`
-* `network_passphrase`: (optional but recommended) Stellar network passphrase used by the server. This allows the client to verify that it's using the correct passphrase when signing and is useful for identifying when a client or server have been configured incorrectly.
+      * `manage_data(source: server account, key: 'web_auth_domain', value: web_auth_domain)`
+        * The source account is the **Server Account**
+        * The value is the **Server**'s domain. It can be at most 64 characters.
+      * zero or more `manage_data(source: server account, ...)` reserved for future use
+  * signature by the **Server Account**
+* `network_passphrase`: (optional but recommended) Stellar network passphrase used by the **Home Domain**. This allows a **Client** to verify that it's using the correct passphrase when signing and is useful for identifying when a **Client** or **Server** have been configured incorrectly.
```

Comment:
❗ I think this still needs to say that the intention is to check what the server is using, not the home domain. If the client wanted to check what the home domain used, it'd look at the toml.
```suggestion
* `network_passphrase`: (optional but recommended) Stellar network passphrase used by the **Server**. This allows a **Client** to verify that it's using the correct passphrase when signing and is useful for identifying when a **Client** or **Server** have been configured incorrectly.
```